ably/auth_integration_test.go

```diff
@@ -124,51 +113,44 @@ func TestAuth_TokenAuth(t *testing.T) {
 	defer safeclose(t, app)
 
 	beforeAuth := time.Now().Add(-time.Second)
-	if _, err := client.Time(context.Background()); err != nil {
-		t.Fatalf("client.Time()=%v", err)
-	}
-	if _, err := client.Stats().Pages(context.Background()); err != nil {
-		t.Fatalf("client.Stats()=%v", err)
-	}
+	_, err := client.Time(context.Background())
+	assert.Nil(t, err,
```

Comment:
I think it should be assert.NoError when checking an error, functionally the same but describes what is being checked a little better

Reply:
Thank you @Peter-Maguire I agree and have pushed these changes.